promptxchange-frontend/public/manifest.json

```diff
@@ -0,0 +1,25 @@
+{
+  "short_name": "React App",
```

Comment:
Feel free to do this in a later PR, but let's update these generated names (short_name, name, etc.) anywhere they're auto-generated in the project.

Reply:
Done